server/src/main/java/io/druid/segment/realtime/plumber/RealtimePlumber.java

```diff
@@ -363,7 +363,9 @@ private Sink getSink(long timestamp)
                                           }
                                         }
                                     )
-                                )
+                                ),
+                                "query/wait/time",
+                                ImmutableMap.of("segment", theSink.getSegment().getIdentifier())
```

Comment:
this isn't query/wait/time

query/wait/time is the amount of time a segment spends in the priority queue before it is scanned. This metric is measuring the time to scan each individual mini-segment that is created by realtime indexing before handoff